app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ] [OPTIONAL] You may try to explicitly tell the user model (table) that, the author is the user by doing something like this:

```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```